versions.properties

```diff
@@ -9,85 +9,50 @@
 ####
 #### NOTE: Some versions are filtered by the rejectVersionIf predicate. See the settings.gradle.kts file.
 
-# Project versioning, run the following gradle command to update version numbers automatically:
```

Comment:
Unrelated but has been bothering me: I organized and alphabetized the versions, and I added the _version_ property for `version.androidx.compose.compiler=1.5.10` that's always causing annoyance in composite test app builds. This does NOT add compose as a dependency, only sets a version property for when the SDK + test app are used in composite mode.

Reply:
I really shouldn't cross purposes of PRs, i'll pull this out